ci/staging-acceptance-tests.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# These are destructive acceptance tests which should only be run in STAGING, there are explicit checks for this, do not override them
+# Requires a user with cloud_controller.admin access to run since it will be creating/deleting users, organizations and org quotas
+# These are destructive to the `sandbox-fedramp`.  This org was chosen since it: wasn't in use, we have email access to the domain, should remain in the CSV file maintaining the list of verified gov agencies
+
+# Log into CF as an admin
+echo "Logging into ${CF_API} as user ${CF_ADMIN_USER}..."
+cf login -a ${CF_API} -u ${CF_ADMIN_USER} -p "${CF_ADMIN_PASSWORD}" -o cloud-gov -s bots >/dev/null 2>&1
+
+# Confirm that we're targeting the correct API
+api_target=$(cf api | grep -i 'API endpoint' | awk '{print $3}')
+if [[ "$api_target" != *"fr-stage"* ]]; then
+  echo "### THIS IS A DESTRUCTIVE TEST to the sandbox-fedramp org, DO NOT RUN IN PRODUCTION ###"
+  echo "Error: Not targeting staging. Current API endpoint: $api_target, exiting for your own safety."
+  exit 1
+fi
+
+# Create a random 32-character password with hyphens
+PASSWORD=$(cat /dev/urandom | base64 | tr -dc '0-9a-zA-Z' | head -c32)
+
+# Create the user
+echo "Creating CF user test.user@fedramp.gov with a random password..."
+cf create-user "test.user@fedramp.gov" "$PASSWORD" >/dev/null 2>&1
+
+# Sleep for 45 seconds, app runs every 30 seconds
+echo "Sleeping test for 45 seconds to wait for sandbox-bot app to process the new user..."
+sleep 45
```

Comment:
I have a slight preference for scripts that retry for X seconds than an arbitrary sleep. Because the time could actually be less than the arbitrary sleep. But not going to insist on it

Reply:
Retry logic has been added, I like this suggestion!  The new version can be seen with https://ci.fr.cloud.gov/teams/main/pipelines/cg-sandbox-bot/jobs/run-sandbox-bot-stage-acceptance-tests/builds/5

Reply:
Awesome work!